src/backend/distributed/executor/repartition_join_execution.c

```diff
@@ -63,6 +65,8 @@ ExecuteDependentTasks(List *topLevelTasks, Job *topLevelJob)
 
 	List *allTasks = TaskAndExecutionList(topLevelTasks);
 
+	EnsureNoVisibilityProblem(allTasks);
```

Comment:
This name tell me little about what this function does. Maybe EnsureCompatibleLocalExecutionState.